lib/benchee/formatters/unit/count.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Benchee.Unit.Count do
+  alias Benchee.Unit.Common
+
+  @behaviour Benchee.Unit
+
+  @one_billion 1_000_000_000
+  @one_million 1_000_000
+  @one_thousand 1_000
+
+  @units %{
+    billion:  %{ magnitude: @one_billion, short: "B", long: "Billion"},
+    million:  %{ magnitude: @one_million, short: "M", long: "Million"},
+    thousand: %{ magnitude: @one_thousand, short: "K", long: "Thousand"},
+    one:      %{ magnitude: 1, short: "", long: ""},
+  }
```

Comment:
`{}` placement is a bit inconsistent here, the beginning has a space while the end doesn't. Either way is fine with me :)